src/main/groovy/grails/plugin/sentry/SpringSecurityUserEventBuilderHelper.groovy

```diff
@@ -0,0 +1,62 @@
+package grails.plugin.sentry
+
+import com.getsentry.raven.event.EventBuilder
+import com.getsentry.raven.event.helper.EventBuilderHelper
+import com.getsentry.raven.event.interfaces.UserInterface
+
+import javax.servlet.http.HttpServletRequest
+
+/**
+ * @author <a href='mailto:alexey@zhokhov.com'>Alexey Zhokhov</a>
+ */
+class SpringSecurityUserEventBuilderHelper implements EventBuilderHelper {
+
+    static List<String> ipHeaders = ['X-Real-IP',
+                                     'Client-IP',
+                                     'X-Forwarded-For',
+                                     'Proxy-Client-IP',
+                                     'WL-Proxy-Client-IP',
+                                     'rlnclientipaddr']
+
+    def springSecurityService
+    def ravenServletRequestListener
+
+    @Override
+    void helpBuildingEvent(EventBuilder eventBuilder) {
+        def currentUser = springSecurityService?.getCurrentUser()
+
+        if (currentUser) {
+            def SpringSecurityUtils = Class.forName('grails.plugin.springsecurity.SpringSecurityUtils')
+
+            def securityConfig = SpringSecurityUtils.securityConfig
+
+            String usernamePropertyName = securityConfig.userLookup.usernamePropertyName
+            String emailPropertyName = securityConfig.userLookup.emailPropertyName
```

Comment:
For me this `emailPropertyName` returns `[:]` (not `null`) which causes a`MissingPropertyException` exception when trying to get the `currentUser[emailPropertyName]`, I don't belive this property isn't a default one in spring security core plugin. Maybe instead provide a `Map` that can be iterated over for the properties someone wishes to include? It would allow people with custom `User` and custom `Principal`s to include custom data they have.